### PR TITLE
[DEVHUB-1280] Not Able to Update Live Content - Fix tagging and pull policy for triggered builds

### PR DESCRIPTION
**Pull request by ajhenri**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -174,7 +174,7 @@ steps:
       add_repos: [mongodb=https://10gen.github.io/helm-charts]
       namespace: devrel
       release: devcenter-frontend
-      values: image.tag=git-${DRONE_COMMIT_SHA:0:7},image.repository=795250896452.dkr.ecr.us-east-1.amazonaws.com/devrel/${DRONE_REPO_NAME},ingress.enabled=true,ingress.hosts[0]=devcenter-frontend.devrel.prod.corp.mongodb.com
+      values: image.tag=latest-prod,image.pullPolicy=Always,image.repository=795250896452.dkr.ecr.us-east-1.amazonaws.com/devrel/${DRONE_REPO_NAME},ingress.enabled=true,ingress.hosts[0]=devcenter-frontend.devrel.prod.corp.mongodb.com
```
Comment (mcolella14) [inline .drone.yml:177]: Would rollbacks still work if we point to `latest-prod`? Or would we just like manually tag the image we want to roll back to as `latest-prod` before rolling back?

**Review by mcolella14 — APPROVED** [review]
LGTM!

**Comment by ajhenri** [comment]
@mcolella14 This isn't finished yet, didn't realize this wasn't in draft.

File: .drone.yml (hunk 1 above)
Comment (ajhenri) [reply .drone.yml:177]: No, I'm still working on this.

**Review by mcolella14 — APPROVED** [review]

**Comment by ajhenri** [comment]
Thanks @mcolella14 @harikakotipalli ! Going to merge this in to staging and if all goes well, will do a prod deploy in the AM tomorrow.

**Pull request merged**